src/Controls/tests/DeviceTests/Elements/TextInput/TextInputFocusTests.Android.cs

```diff
@@ -11,7 +11,7 @@ public abstract partial class TextInputFocusTests<THandler, TView> : ControlsHan
 		where THandler : class, IViewHandler, IPlatformViewHandler, new()
 		where TView : VisualElement, ITextInput, new()
 	{
-		[Fact]
+		[Fact(Skip = "It sometimes fails in CI and we have added an UITest")]
```

Comment:
If we're moving this to appium we can just delete it here

Reply:
Removed.